app/controllers/reports_controller.rb

```diff
@@ -4,9 +4,13 @@ class ReportsController < ApplicationController
   require_permission :can_view_reports?
 
   def index
-    @org_by_county = {}
-    Organization.counties.sort_by { |county| (county.presence || "no county").downcase }.each do |county_name|
-      @org_by_county[county_name.presence || "No County"] = Organization.where(county: county_name)
+    # raise params.inspect
+    @report = params[:report]
+    if @report == "value_by_county"
+      @org_by_county = Organization.by_county_report
+    elsif @report == "total_inventory_value"
+      @category = Category.find(params[:category_id]) if params[:category_id]
```

Comment:
I see now that the sidebar uses all categories always, so that wouldn't quite have the desired behavior... though I think it might be reasonable to have the view simply call `Category.all` in the sidebar, and keep @categories for just the results. These queries aren't going to be big enough to cause any performance problem, and it would simplify the view logic.